### PR TITLE
minor fix in controlnet flax example

### PR DESCRIPTION
**Pull request by yiyixuxu**
this PR:
1. add the step to install `wandb` in step by step guide (since it's not in the requirement file)
2. fix a bug: currently the script will fail if you choose `push_to_hub` not run `log_validation `

### CODE REVIEW

File: examples/controlnet/README.md [hunk 1]
```diff
@@ -320,6 +320,12 @@ Then cd in the example folder and run
 pip install -U -r requirements_flax.txt
 ```
 
+If you want to use Weights and Bias logging, you should also install `wandb` now
```
Comment (sayakpaul) [inline examples/controlnet/README.md:323]: It's Weights and Biases.

**Review by sayakpaul — APPROVED** [review]
Thanks for fixing this!

File: examples/controlnet/train_controlnet_flax.py [hunk 2]
```diff
@@ -213,6 +214,17 @@ def parse_args():
...
+        "--controlnet_revision",
+        type=str,
+        default=None,
+        help="Revision of controlnet model identifier from huggingface.co/models.",
+    )
+    parser.add_argument(
+        "--controlnet_from_pt",
+        action="store_true",
+        help="Load the controlnet model from a PyTorch checkpoint.",
+    )
```
Comment (sayakpaul) [inline examples/controlnet/train_controlnet_flax.py:227]: @yiyixuxu have we talked about these two arguments in the README?

Comment (yiyixuxu) [reply examples/controlnet/train_controlnet_flax.py:227]: no - should we?  the doc string is not enough?

Comment (sayakpaul) [reply examples/controlnet/train_controlnet_flax.py:227]: No strong opinion. 

I think it's fine either way.

**Review by patrickvonplaten — APPROVED** [review]

**Pull request merged**